projects/ngx-datatable/src/lib/themes/material.scss

```
@@ -466,6 +471,34 @@ $datatable-summary-row-background-hover: #ddd !default;
   }
 }
 
+@keyframes ghost {
+  from {
+    background-position: 0vw 0;
+  }
+  to {
+    background-position: 100vw 0;
+  }
+}
+.ghost-loader {
+  position: sticky;
+  top: 20px;
+  height: 0;
+  background-color: $ngx-datatable-default-background;
+  .line {
+    width: 100%;
+    height: 12px;
+    margin: 0.9rem 1.2rem;
+    background: linear-gradient(90deg, $datatable-ghost-loader-background-light, $datatable-ghost-loader-background, $datatable-ghost-loader-background-light) 0 0/ 80vh 100% fixed;
+    background-color: $datatable-ghost-loader-background-dark;
+    animation: ghost 750ms infinite linear;
+    
+  }
+
+  .ghost-element {
+      display: flex;
+  }
+}
```

- Might make sense to add a chapter comment here.
- Slightly more consistent indents and spacing.

```suggestion
/**
 * Ghost loader animation
 */
@keyframes ghost {
  from {
    background-position: 0vw 0;
  }
  to {
    background-position: 100vw 0;
  }
}

.ghost-loader {
  position: sticky;
  top: 20px;
  height: 0;
  background-color: $ngx-datatable-default-background;

  .line {
    width: 100%;
    height: 12px;
    margin: 0.9rem 1.2rem;
    background: linear-gradient(90deg, $datatable-ghost-loader-background-light, $datatable-ghost-loader-background, $datatable-ghost-loader-background-light) 0 0/ 80vh 100% fixed;
    background-color: $datatable-ghost-loader-background-dark;
    animation: ghost 750ms infinite linear;
  }

  .ghost-element {
    display: flex;
  }
}
```